release-notes/changes-in-timescaledb-2.md

```diff
@@ -320,6 +320,45 @@ After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due
 continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the 
 desired interplay. 
 
+Another change in 2.0 is that `drop_chunks` and the retention policy will no longer 
+automatically refresh continuous aggregates to account for changes in raw data 
+after the last refresh. Previously, the goal was to ensure that all updates were processed 
+prior dropping. However, it often didn't work as intended.
```

Comment:
```suggestion
Another change in 2.0 is that invoking `drop_chunks`, either manually or through a retention policy, will no longer 
refresh continuous aggregates for the interval represented by the chunk. Previously, TimescaleDB 1.7 would attempt to materialize any modifications in raw chunk data into the continuous aggregate before the chunk was dropped. In practice, it often didn't work as intended.
```
Based on feedback from Erik, suggesting just a few tweaks. Mostly it wasn't clear as written (to me) that `drop_chunks` was used in both manual and policy updates here.  Take or leave, just suggesting.

Reply:
@ryanbooz Retention policy doesn't call SQL function `drop_chunk`, but both calls the same internal function. I think you are missing that both are performing chunk drops.
I will see how incorporate it better into the text.

Reply:
Gotcha. Main thing I was trying to clarify in context is that this section is about retention policies, which in truth, we never specifically mention that it's dropping chunks. Obviously, it's implied... but I'm just now realizing that as I re-read.  :-)

Anyway, it feels like just a little more context/setup would be helpful.  Maybe something like...

```suggestion
One other change in 2.0 impacts how data in continuous aggregates is updated. When dropping older data from a hypertable, either manually when calling `drop_chunks` or with a data retention policy, the continuous aggregate will no longer be refreshed for the interval represented by the chunk....
```

Reply:
I made minor change to say dropping chunks, but haven't followed suggestion completely as it increases the text. I feel it is not the best place to go into much implementation details.